### PR TITLE
fixed proximity mine effect to match errata

### PR DESCRIPTION
**Pull request by davidhorm**
Modified the effect text to match the errata as of rules v1.0.2 (Sept 19) https://images-cdn.fantasyflightgames.com/filer_public/2a/57/2a57fdb2-e13f-4d58-b9db-bb082dd23dcc/swzrulesreference_v102.pdf

### CODE REVIEW

File: data/upgrades/device.json [hunk 1]
```diff
@@ -82,7 +82,7 @@
         "device": {
           "name": "Proximity Mine",
           "type": "Mine",
-          "effect": "After a ship overlaps or moves through this device, it detonates. When this device detonates, that ship rolls 2 attack dice. That ship then suffers 1 [Hit]/[Critical Hit] damage for each matching result."
+          "effect": "After a ship overlaps or moves through this device, it detonates. When this device detonates, that ship rolls 2 attack dice. That ship then suffers 1 [Hit] plus 1 [Hit]/[Critical Hit] damage for each matching result."
```
Comment (guidokessels) [inline data/upgrades/device.json]: I think this should read:

```suggestion
          "effect": "After a ship overlaps or moves through this device, it detonates. When this device detonates, that ship rolls 2 attack dice. That ship then suffers 1 [Hit] damage plus 1 [Hit]/[Critical Hit] damage for each matching result."
```

Comment (davidhorm) [reply data/upgrades/device.json]: But if I add that extra "damage" text then it won't exactly match the errata text. Notice in the snippet below that "damage" is only written once.

proxmine

Comment (guidokessels) [reply data/upgrades/device.json]: Ah, I was looking at this part:

proximity-mines

Seems like FFG isn't consistent in the same document ;)

**Comment by davidhorm** [comment]
Didn't realize that FFG also applied their "errata" text on page 20 of that rules reference. I'll go with that version instead because that's probably the page that everybody else is going to print out.

**Pull request merged**